### PR TITLE
ACTIVECOM-1606 Fix webpage link changed to https issue

### PR DESCRIPTION
**Pull request by marvin-min**
#### JIRA
- https://jirafnd.dev.activenetwork.com/browse/ACTIVECOM-1606?filter=34274

#### Test link:
- http://a3coreint10.dev.activenetwork.com/running/articles/10-healthy-ish-ice-cream-recipes-for-runners
- http://a3coreint10.dev.activenetwork.com/fitness/articles/stocking-stuffers-for-athletes

### CODE REVIEW

**Comment by RogerTongTong** [comment]
On it

**Comment by marvin-min** [comment]
@RogerTongTong Exactly, Just checked at A3 side. `imageName =  'MediaGalleryButton'` should be a good decision.

File: lib/actv/asset.rb [hunk 1]
```diff
@@ -134,8 +134,7 @@ def channels
 
     def images
       @images ||= Array(@attrs[:assetImages]).map do |img|
-        img[:imageUrlAdr] = replace_http_to_https img[:imageUrlAdr]
-        img[:linkUrl] = replace_http_to_https img[:linkUrl]
```
Comment (hlcfan) [inline lib/actv/asset.rb]: `linkUrl` doesn't need to be updated?

Comment (marvin-min) [reply lib/actv/asset.rb]: I think yes~

Comment (hlcfan) [reply lib/actv/asset.rb]: Any reason not to?

Comment (marvin-min) [reply lib/actv/asset.rb]: What we just want keep the resource stuff(image/video/js/css) changed. for the link stuff, we do not want them chaned.

**Comment by RogerTongTong** [comment]
Look great 👍

**Pull request merged**